### ISSUE TITLE
Basic packsquash TOML does not work.

### ISSUE THREAD

**Issue by DragonsAscent** [ComunidadAylas/PackSquash#17]
**Describe the bug**
Using the TOML file from the wiki I get all sorts of errors regarding various strings such as `["**/*.png"]`, `["assets/*/sounds/{music,ambience}/?*.{og[ga],mp3,wav,flac}"]`, and then it gives me "An error occured while parsing the settings file at ... invalid type: integer "64", expected struct File at line 1 column 1" when its just a commented area?

**To Reproduce**
1. Load basic TOML file from wiki example
2. Remove areas that are providing errors
3. Get stuck at the last error about line 1 column 1

**Expected behavior**
Removing the comments at line 1 - 7 will fix the TOML file and allow me to squash my pack.

**Attached files**
https://gist.github.com/DragonsAscent/dc8a0f3023c33d3869470b678b47f22f

**Additional context**
Running on windows 10 with everything installed properly.

**Comment by AlexTMjugador** [comment]
Thanks for reporting this issue!

I've just tried PackSquash v0.2.1 (the latest release version) with the settings file you provided, and indeed I receive an error similar to yours (although in my case it points at the line 26, but the actual error neither occurs on the line 1 or 26, so I guess it's just a confusing error message returned by the TOML deserializer):

```
Reading settings from packsquash.toml...
An error occurred while parsing the settings file at packsquash.toml: invalid type: integer `64`, expected struct File at line 26 column 1
```

The actual problem is that the deserializer expected a struct somewhere in the file, but instead it received a integer whose value is 64, and the only setting that is set to 64 in the provided settings file is `output_file_spooling_buffer_size`, which is **not** implemented in the latest release version, but will be available on the upcoming v0.3.0 version. So the most obvious problem I can see with the example TOML file in the wiki is that it includes a setting that PackSquash will currently reject, which should be removed or commented out. I removed the first 7 comment lines as you mentioned and, as expected, they didn't make PackSquash accepted the file or not.

In particular, this settings file works fine for me, which is exactly the same one as you provided, but with that unimplemented setting commented out: https://pastebin.com/HPwWXccH. Please confirm that whether it works fine for you too.

Anyway, to reduce confusion and make the examples actually work out of the box without reading the documentation in detail (you're not the first one to be bitten by this quirk), I'll remove the `output_file_spooling_buffer_size` setting from the examples. It was there so I didn't miss out documenting it when v0.3.0 got released, but it's taking a bit and this issue will do to remind me about that setting, so it no longer belongs in the examples.

**Comment by DragonsAscent** [comment]
This worked, thanks!
